x-pack/plugins/security_solution/public/management/pages/policy/view/components/policy_endpoint_link.tsx

```diff
@@ -54,6 +58,9 @@ export const PolicyEndpointListLink = memo<
   }, [getAppUrl, filterByPolicyQuery, search]);
   const clickHandler = useNavigateByRouterEventHandler(toRoutePathWithBackOptions);
 
+  if (nonLinkCondition) {
```

Comment:
I'm ok with that approach but I'm still thinking it is weird (at least to me) that a component with the word `Link` on his name can return a simple text depending on a prop. But lets keep this as it is if that make sense 🙂